main/provider/proxy.ts

```diff
@@ -0,0 +1,22 @@
+import log from 'electron-log'
+import { EventEmitter } from 'stream'
+
+// @ts-ignore
+import EthereumProvider from 'ethereum-provider'
+
+class ProviderProxyConnection extends EventEmitter {
+  async send (payload: RPCRequestPayload) {
+    try {
+      this.emit('provider:send', payload)
```

Comment:
I think this try/catch is left over from the other connection class, don't think we can throw here